README.md

```diff
@@ -54,248 +99,292 @@ $(SRCROOT)/Carthage/Build/iOS/CryptoSwift.framework
-**Parameters**<br/>
-&nbsp;_userId: OST Platform user id provided by application server_<br/>
-&nbsp;_pin: User Pin_<br/>
-&nbsp;_passphrasePrefix: Passphrase prefix provided by application server_<br/>
-&nbsp;_spendingLimit: Spending limit in a transaction in atto BT_<br/>
-&nbsp;_expireAfterInSec: Session key validat duration_<br/>
-&nbsp;_delegate: Callback implementation object for application communication_<br/>
+### 3. activateUser
+It `authorizes` the registered device and activates the user. User activation deploys  **TokenHolder**, Device manager  contracts on blockchain. Session keys are also created and authorized during `activateUser` workflow. So after `user activation`, users can perform wallet actions like executing transactions and reset pin. 
 
-```Swift
+```
 OstWalletSdk.activateUser(
     userId: String,
     userPin: String,
     passphrasePrefix: String,
     spendingLimit: String,
     expireAfterInSec: TimeInterval,
     delegate: OstWorkflowDelegate
-    )
+)
 ```
 
-### Authorize session
-A session is a period of time during which a sessionKey is authorized to sign transactions under a pre-set limit on behalf of the user.
+| Parameter | Description |
+|---|---|
+| **userId** <br> **String**	| Unique identifier of the user stored in OST Platform  |
+| **userPin** <br> **String**	| User's PIN created during wallet setup.|
+| **passphrasePrefix** <br> **String**	| A constant unique identifier for your user. |
+| **spendingLimit** <br> **String**	| Spending limit of session key in [atto BT](/platform/docs/sdk/guides/execute_transaction/#converting-brand-token-to-atto-brand-token).  |
+| **expireAfterInSec** <br> **TimeInterval**	| Expire time of session key in seconds. |
+| **delegate** <br> **OstWorkflowDelegate**	| An object that implements the callback function available in `OstWorkflowDelegate` protocol. These callback functions are needed for communication between app and wallet SDK. Implement `flowComplete` and `flowInterrupt` callback functions to get the workflow status. Details about other callback function can be found in [OstWorkflowDelegate protocol reference](/platform/docs/sdk/references/wallet_sdk/iOS/latest/protocols/).  |
 
-The device manager, which controls the tokens, authorizes sessions. <br/><br/>
-**Parameters**<br/>
-&nbsp;_userId: OST Platform user id provided by application server_<br/>
-&nbsp;_spendingLimit: Spending limit in a transaction in atto BT_<br/>
-&nbsp;_expireAfterInSec: Session key validat duration_<br/>
-&nbsp;_delegate: Callback implementation object for application communication_<br/>
 
-```Swift
+
+<br>
+### 4. addSession
+This workflow will create and authorize the session key that is needed to do the transactions. This flow should be called if the session key is expired or not present. 
+
+```
 OstWalletSdk.addSession(
     userId: String,
     spendingLimit: String,
-    expireAfterInSec: TimeInterval,
+    expiresAfterInSecs: TimeInterval,
```

Comment:
Not sure why the signature changed here.
As per the last release:
https://github.com/ostdotcom/ost-wallet-sdk-ios/blob/release-2.2/OstWalletSdk/Workflows/OstWorkFlowFactory.swift

The method signature is `expireAfterInSec` and not `expiresAfterInSecs`.
Changing this would mean a major version release (i.e. going from 2 to 3) as this would be a breaking change.
Although, this might not impact android.